src/components/patch/PatchContainer.test.tsx

```diff
@@ -31,7 +31,7 @@ describe("PatchContainer", () => {
     const patch = wrapper.findWhere(node => node.key() === "5d126fa93627e070b33dbbc0").find(Patch);
     expect(patch).toHaveLength(1);
     expect(patch.state("description")).toBeDefined();
-    expect(patch.state("description")).toBe("Patch from domino.weir at Tuesday, June 25, 2019 3:02 PM on project spruce");
```

Comment:
This was causing my patch to fail in Evergreen because I made an assumption about the locale it would be running from- (expected time was 3:02 PM, actual time was 7:02 PM). The main point of this test is to make sure a description is defined and nonempty for every patch, so instead of looking for an exact match I'm just checking that the patch description contains the text that it should start with.